src/main/resources/initialize.sql

```diff
@@ -0,0 +1,447 @@
+        REFERENCES public."OnlineDonor" ("OnlineDonorID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."ODD_Makes"
+    OWNER to postgres;
+
+
+--14 Table: public.AS_Makes
+
+-- DROP TABLE IF EXISTS public."AS_Makes";
+
+CREATE TABLE IF NOT EXISTS public."AS_Makes"
+(
+    "AdminID" integer NOT NULL DEFAULT nextval('"AS_Makes_AdminID_seq"'::regclass),
+    "SessionID" bigint NOT NULL DEFAULT nextval('"AS_Makes_SessionID_seq"'::regclass),
+    CONSTRAINT "AS_Makes_pkey" PRIMARY KEY ("AdminID", "SessionID"),
+    CONSTRAINT "AS_Makes_AID_fkey" FOREIGN KEY ("AdminID")
+        REFERENCES public."Admin" ("AdminID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "AS_Makes_SID_fkey" FOREIGN KEY ("SessionID")
+        REFERENCES public."Session" ("SessionID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."AS_Makes"
+    OWNER to postgres;
+
+
+--15 Table: public.AM_Assigns
+
+-- DROP TABLE IF EXISTS public."AM_Assigns";
+
+CREATE TABLE IF NOT EXISTS public."AM_Assigns"
+(
+    "AdminID" integer NOT NULL DEFAULT nextval('"AM_Assigns_AdminID_seq"'::regclass),
+    "MentorID" integer NOT NULL DEFAULT nextval('"AM_Assigns_MentorID_seq"'::regclass),
+    CONSTRAINT "AM_Assigns_pkey" PRIMARY KEY ("AdminID", "MentorID"),
+    CONSTRAINT "AM_Assigns_AID_fkey" FOREIGN KEY ("AdminID")
+        REFERENCES public."Admin" ("AdminID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "AM_Assigns_MID_fkey" FOREIGN KEY ("MentorID")
+        REFERENCES public."Mentor" ("MentorID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."AM_Assigns"
+    OWNER to postgres;
+
+
+--16 Table: public.SMTT_Takes
+
+-- DROP TABLE IF EXISTS public."SMTT_Takes";
+
+CREATE TABLE IF NOT EXISTS public."SMTT_Takes"
+(
+    "SessionID" bigint NOT NULL DEFAULT nextval('"SMTT_Takes_SessionID_seq"'::regclass),
+    "MenteeID" integer NOT NULL DEFAULT nextval('"SMTT_Takes_MenteeID_seq"'::regclass),
+    CONSTRAINT "SMTT_Takes_pkey" PRIMARY KEY ("SessionID", "MenteeID"),
+    CONSTRAINT "SMTT_Takes_MID_fkey" FOREIGN KEY ("MenteeID")
+        REFERENCES public."Mentee" ("MenteeID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "SMTT_Takes_SID_fkey" FOREIGN KEY ("SessionID")
+        REFERENCES public."Session" ("SessionID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."SMTT_Takes"
+    OWNER to postgres;
+
+
+--17 Table: public.FS_Has
+
+-- DROP TABLE IF EXISTS public."FS_Has";
+
+CREATE TABLE IF NOT EXISTS public."FS_Has"
+(
+    "FeedbackID" bigint NOT NULL DEFAULT nextval('"FS_Has_FeedbackID_seq"'::regclass),
```

Comment:
same question with the default autoincrement. I think default nextval ... should be removed. right?